<div class="container">
  <div class="row">
    <div class='mx-auto col-11 col-sm-10 col-lg-8 col-xl-6'>
      <div class="mx-auto col-5">
      <h3 class="mt-3 text-center">配送先の編集</h3>
        <% if @address.errors.any? %>
          <div id="error_explanation">
            <h3><%= pluralize(@address.errors.count, "error") %>　件のエラーが発生しました。</h3>
            <ul>
              <% @address.errors.full_messages.each do |message| %>
                <li><%= message %></li>
              <% end %>
            </ul>
          </div>
        <% end %>
        <%= form_with model: @address, url: public_order_address_path do |f| %>
          <%= f.label "ご配送先の宛名", class: "form-label" %>
            <%= f.text_field :order_name, class: "form-control" %>

          <%= f.label "郵便番号", class: "form-label mt-2" %>
            <%= f.text_field :post, class: "form-control" %>

          <%= f.label "ご配送先の住所", class: "form-label mt-2" %>
            <%= f.text_area :address, class: "form-control" %>

          <%= f.submit "更新", class: "btn btn-outline-success btn-sm btn-block mt-2" %>
        <% end %>
      </div>
    </div>
  </div>
</div>